samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -326,8 +326,10 @@ - (UIImage*)transformImage:(UIImage*)image {
     // create CGImageRef from vImage_Buffer output
     CGColorSpaceRef colorSpace = CGColorSpaceCreateDeviceRGB();
 
-    CGContextRef ctx = CGBitmapContextCreate(
-        outBuffer.data, outBuffer.width, outBuffer.height, 8, outBuffer.rowBytes, colorSpace, (CGBitmapInfo)kCGImageAlphaNoneSkipLast);
+    const CGBitmapInfo bitmapInfo = kCGImageAlphaNoneSkipLast | kCGBitmapByteOrder32Little;
```

Comment:
Understood, this code however didn't need kCGBitmapByteOrder32Little before, so this will break apps that don't specify it?

---
In reply to: [71218160](https://github.com/Microsoft/WinObjC/pull/641#discussion_r71218160) [](ancestors = 71218160)